src/server/src/main/docker/entrypoint.sh

```diff
@@ -25,7 +25,7 @@ if [ "$1" = 'cassandra-reaper' ]; then
     /usr/local/bin/configure-metrics.sh
     exec java \
             ${JAVA_OPTS} \
-            -jar /usr/local/lib/cassandra-reaper.jar server \
+            -cp "/usr/local/lib/*" io.cassandrareaper.ReaperApplication server \
```

Comment:
@leleueri, could you please explain why precisely we need this change? I'm not able to work out how it helps.

Reply:
Hi @rzvoncek,

This change has been introduced by the commit https://github.com/thelastpickle/cassandra-reaper/commit/35b5d59ba5572f3adebd02ba8352e63b426c3e3a made by Vincent, he may have encountered an issue with the shaded jar?

I'm not involved in Elassandra anymore, could you please ask to @vroyer ?

Regards,
Eric

Reply:
Hi @rzvoncek,
This allows to add custom jars to the classpath by dropping a jar file into the /usr/local/lib (with a Dockerfile COPY x /usr/local/lib) without rewriting the entrypoint.sh. Thus, anyone can easily customize its own image from the reaper one.

Reply:
Thanks for clarifying both contact points and the change. It all makes sense.
I'm work on getting #885 in, then come back to this.